internify/src/assets/Populations/studentScript.py

```diff
@@ -0,0 +1,293 @@
+import json
```

Comment:
@tommytwm  @ReneReid  @aleemer do we want this in the assets folder? My understanding is assets should only be for assets like images.

Reply:
Talked to @ReneReid about this. This was supposed to be a temporary measure until we had mongo set up-We eventually wanna move all of this data to our DB. So since we have that set up now, we should move this to our DB instead.